### DISCUSSION TITLE
Welcome to Shuffle-Frontend-Developer-Software-Engineering Discussions!

### DISCUSSION THREAD

**Discussion by ashwinexe** [GitHub-Octernships/Shuffle-Frontend-Developer-Software-Engineering#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by manthan0x23** [comment]
@ashwinexe Hey there sir! , I have completed the assignment (frontend-octernship) and submitted it the way posted Can you please check and review it once ! Thank you so much .

**Reply by President10000** [reply]
Can you tell me where i have to push my repository

**Reply by sachdevavaibhav** [reply]
You must have got a private repo with your name.

**Reply by manthan0x23** [reply]
Make a private repo from template and push your code there

**Reply by President10000** [reply]
Do  i have to push only updated files or all frontend  files

**Reply by sachdevavaibhav** [reply]
The updated ones while maintaining the folder structure.

**Reply by ashwinexe** [reply]
> Make a private repo from template and push your code there

Hey there @manthan-sharma-23, We do not accept private submissions. Only submission repositories created when applying via [Octernships dashboard](https://education.github.com/students/octernships) are allowed

**Reply by manthan0x23** [reply]
Under GIthubOcternships right the submission repo created ....I meant that only .

**Comment by maverox** [comment]
i am unable to make post requests on shuffle's endpoint programmatically from my react component,
getting error 401

**Reply by maverox** [reply]
but it is working when the same request is made through ui from somewhere else in codebase
